File: andre/examples/sum-variations/sum.py
# Python sum program

def sum(seq):
    s=0
    for v in seq:
        s+=v
    return s

seq=[5,6,1,8,3,7]

print(sum(seq))

def sum(seq, n):
    if (n <= 0):
        return 0
    return seq[n - 1] + sum(seq, n - 1)

print(sum(seq, 6))

File: buff/classes/354/pub/sum/zig/sum.zig
// Zig sum program

const std=@import("std");
const stdout=std.io.getStdOut().writer();

pub fn sum(seq: []u32) u32 {
    var s: u32=0;
    for (seq) |v|
        s+=v;
    return s;
}

pub fn main() !void {
    var seq=[_]u32{5,6,1,8,3,7};
    try stdout.print("{d}\n",.{sum(&seq)});
}
